### PR TITLE
Change the meaning of `Flush` on cluster mode

### PR DESCRIPTION
**Pull request by HTHou**
`Flush` command in cluster should flush data on all data nodes.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/FlushTask.java [hunk 1]
```diff
@@ -54,10 +54,10 @@ public ListenableFuture<ConfigTaskResult> execute(IConfigTaskExecutor configTask
       tFlushReq.setIsSeq(flushStatement.isSeq().toString());
     }
     IoTDBConfig config = IoTDBDescriptor.getInstance().getConfig();
-    if (flushStatement.isLocal()) {
-      tFlushReq.setDataNodeId(config.getDataNodeId());
-    } else {
+    if (flushStatement.isCluster()) {
       tFlushReq.setDataNodeId(-1);
+    } else {
+      tFlushReq.setDataNodeId(config.getDataNodeId());
```
Comment (ericpai) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/FlushTask.java:60]: If SQL is `FLUSH` in standalone mode, then `flushStatement.isCluster() == true` in your ASTVisitor modificaton. So what will happen here?

Comment (HTHou) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/FlushTask.java:60]: Standalone mode won’t check it, only flush the data in StorageEngine.

Comment (ericpai) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/FlushTask.java:60]: Got it.

**Review by ericpai — APPROVED** [review]

**Comment by ericpai** [comment]
LGTM

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**